pkg/operator/target_config_reconciler.go

```diff
@@ -1254,7 +1254,7 @@ func (c *TargetConfigReconciler) manageConfigMap(descheduler *deschedulerv1.Kube
 		}
 	}
 
-	if descheduler.Spec.ProfileCustomizations != nil && descheduler.Spec.ProfileCustomizations.DevEnableSoftTainter {
+	if descheduler.Spec.ProfileCustomizations != nil && descheduler.Spec.ProfileCustomizations.KubevirtRelieveAndMigrateSoftTainter == deschedulerv1.SoftTainterEnabled {
```

Comment:
Are we going to simply ignore the now deprecated `DevEnableSoftTainter` parameter ?

Reply:
You are absolutely correct here. I haven't noticed this caveat. We will have a parameter that does nothing, that is unfortunate (albeit acceptable IMHO as no one is using it yet).

| DevEnableSoftTainter | KubevirtRelieveAndMigrateSoftTainter  | Action |
| ----- | --- | --- |
| false | "Enabled" | Enables the Soft Tainter |
| true | "Enabled" | Enables the Soft Tainter |
| false  | "Disabled" | Disables the Soft Tainter |
| true  | "Disabled" | Disables the Soft Tainter |
| false | "" | **Not clear what is expected** |
| true  | "" | Enables the Soft Tainter |

Reply:
`DevEnableSoftTainter` is basically ignored, if `KubevirtRelieveAndMigrateSoftTainter` is not specified, the softtainter will be enabled (default option with the RelieveAndMigrate profile).

Reply:
`DevEnableSoftTainter` is an override for case when the soft-tainter is disabled by default. Given it's gonna by enabled by default the field is no-op from now on. So it can be safely ignored.